news update 2021

minguibernardo opened the pull request:
Hi guys, I made some basic changes to the community page, resolving some log errors

Now what’s been added 👍🏼 
.................................................................................
add new font-family
add new footer
add github and facebook footer
add new img
its all.....

minguibernardo committed:
news update

add new font-family
add new footer
add github and facebook footer
add new img
its all.....

braulio94 commented on index.html:
```diff
@@ -1,71 +1,92 @@
+			<p class="lead">Connect with other developers interested in learing Google tecnologies. GDG Luanda welcomes
+				you to attend our meetings.</p>
+			<p class="lead">
+				<a href="https://www.meetup.com/pt-BR/GDG-Luanda/" target="blank" class="btn btn-md btn-light">Join
```
Mudar de o link do meetup para
https://gdg.community.dev/GDG-Luanda/

minguibernardo replied:
A notado  ✔

braulio94 commented on index.html:
```diff
@@ -1,71 +1,92 @@
-			</nav>
-		  </div>
+			<div class="inner">
+				<img src="assets/img/logo.png" class="img-logo masthead-brand" alt="">
```
Substituir o logo antigo pelo actual 
![logo](https://user-images.githubusercontent.com/20057010/107130940-ba1fd000-68d2-11eb-91b7-3c4728a9d794.png)

minguibernardo replied:
A notado  ✔

braulio94 commented on index.html:
```diff
@@ -1,71 +1,92 @@
+
+					<div class="col-md-4 col-sm-6 col-xs-12">
+						<ul class="social-icons">
+							<li><a class="facebook github" href="https://github.com/gdgluanda/gdgluanda.github.io">
```
Se poder remover o link do Github e adicionar o link para o Twitter seria fixe!
https://twitter.com/luandagdg

minguibernardo replied:
A notado  ✔

braulio94 reviewed (changes requested):
Olá Cristovão

Obrigado pela nova cara que deste a pagina da comunidade.
Estavamos mesmo a precisar de uma actualização. No entanto já que estamos aqui pra fazer bem as coisas sugeri mais algumas mudaças e depois fazemos o merge, combinado? Otimo!

minguibernardo committed:
Important Changes

What was added?
Typed Text / a little animation in the text
new logo  / GDG
new links  /meetup and twitter upadate
......................................................................................................................................
some CSS level changes have also been added and JAVASCRIPT

Co-Authored-By: Braulio Cassule <20057010+braulio94@users.noreply.github.com>

minguibernardo committed:
Up

svg logo add comment

Co-Authored-By: Braulio Cassule <20057010+braulio94@users.noreply.github.com>
